lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEF.cs

```diff
@@ -141,10 +165,12 @@ public static void MainSimple(string[] args)
             var runtime = args.Length > 0 ? args[0] : Local;
 
             // Execute the HelloREEF, with these parameters injected
-            TangFactory.GetTang()
-                .NewInjector(GetRuntimeConfiguration(runtime))
-                .GetInstance<HelloREEF>()
-                .Run();
+            var injector = TangFactory.GetTang()
+                .NewInjector(GetRuntimeConfiguration(runtime));
+            string containerRegistryServer = injector.GetNamedInstance<ContainerRegistryServer, string>(GenericType<ContainerRegistryServer>.Class);
+            ISet<int> ports = injector.GetNamedInstance<TcpPortSet, ISet<int>>(GenericType<TcpPortSet>.Class);
```

Comment:
This is related to Azure Batch only and should be kept out of the global logic.

Reply:
Removed this code.

---
In reply to: [201140913](https://github.com/sharathmalladi/reef/pull/5#discussion_r201140913) [](ancestors = 201140913)